src/plugins/plugin.legend.js

```diff
@@ -101,19 +101,41 @@ class Legend extends Element {
 	constructor(config) {
 		super();
 
-		const me = this;
-		helpers.extend(me, config);
+		helpers.extend(this, config);
 
 		// Contains hit boxes for each dataset (in dataset order)
-		me.legendHitBoxes = [];
+		this.legendHitBoxes = [];
```

Comment:
There is a reason for not using `me` in constructor?

Reply:
Yeah, Typescript doesn't figure out that it's the same as `this`. It's the only thing I ran into that I think it's really not handling that well. We could probably file a bug against it